core/unit_test/TestUniqueToken.hpp

```diff
@@ -42,11 +42,11 @@
 //@HEADER
 */
 
-#include <iostream>
+#include <gtest/gtest.h>
 
 #include <Kokkos_Core.hpp>
 
-namespace Test {
+namespace {
```

Comment:
Why did you change this?

Reply:
Because that's what we should have been doing the whole time

Reply:
This avoids potential collisions at link-time